### ISSUE TITLE
Different results with cli and maven starter

### ISSUE THREAD

**Issue by ahi** [dependency-check/DependencyCheck#196]
I have two different outputs when Im use the cli version and the maven version
## cli

If i start the dependency check with:
`.\dependency-check-1.2.8-release\bin\dependency-check.bat --app testProject --scan .\TestProject --out .\testproject-report\cmd`

I get the following result (Showing All Dependencies) :

| Dependency | CPE | GAV | Highest Severity | CVE Count | CPE Confidence | Evidence Count |
| --- | --- | --- | --- | --- | --- | --- |
| test-project-1.0-SNAPSHOT.war |  | test-project:test-project:1.0-SNAPSHOT |  | 0 |  | 3 |
| test-project-1.0-SNAPSHOT.war: axis-1.2.jar | cpe:/a:apache:axis:1.2 | axis:axis:1.2 | Medium | 2 | HIGHEST | 7 |
| test-project-1.0-SNAPSHOT.war: commons-logging-1.1.1.jar |  | commons-logging:commons-logging:1.1.1 |  | 0 |  | 19 |
| test-project-1.0-SNAPSHOT.war: spring-core-3.2.0.RELEASE.jar |  | org.springframework:spring-core:3.2.0.RELEASE |  | 0 |  | 6 |
## maven

and if i start it with maven Showing All Dependencies:
`mvn --version` 
Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T18:29:23+01:00)
Maven home: C:\Users\a.kiyak\ws\apache-maven-3.2.5\bin..
Java version: 1.8.0_31, vendor: Oracle Corporation
Java home: C:\Program Files\Java\jdk1.8.0_31\jre
Default locale: de_DE, platform encoding: Cp1252
OS name: "windows 8", version: "6.2", arch: "amd64", family: "dos"

`mvn clean package`

| Dependency | CPE | GAV | Highest Severity | CVE Count | CPE Confidence | Evidence Count |
| --- | --- | --- | --- | --- | --- | --- |
| axis-1.2.jar | cpe:/a:apache:axis:1.2 | axis:axis:1.2 | Medium | 2 | HIGHEST | 10 |
| commons-logging-1.1.1.jar |  | commons-logging:commons-logging:1.1.1 |  | 0 |  | 19 |
| spring-core-3.2.0.RELEASE.jar | cpe:/a:springsource:spring_framework:3.2.0 | org.springframework:spring-core:3.2.0.RELEASE | Medium | 5 | HIGHEST | 15 |
## Result

As you can see, I get CVEs for `spring-core-3.2.0.RELEASE.jar` when maven is used.

My pom.xml

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>

    <groupId>test-project</groupId>
    <artifactId>test-project</artifactId>
    <packaging>war</packaging>
    <version>1.0-SNAPSHOT</version>
    <properties>
        <spring.version>3.2.0.RELEASE</spring.version>
    </properties>
    <dependencies>
        <dependency>
            <groupId>org.springframework</groupId>
            <artifactId>spring-core</artifactId>
            <version>3.2.0.RELEASE</version>
        </dependency>
        <dependency>
            <groupId>axis</groupId>
            <artifactId>axis</artifactId>
            <version>1.2</version>
        </dependency>

    </dependencies>
    <build>
        <plugins>
            <plugin>
                <groupId>org.owasp</groupId>
                <artifactId>dependency-check-maven</artifactId>
                <version>1.2.8</version>
                <configuration>
                    <logFile>target/dependency-check-log.txt</logFile>
                </configuration>
                <executions>
                    <execution>
                        <goals>
                            <goal>check</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
        </plugins>
    </build>

</project>
```

**Comment by jeremylong** [comment]
Thanks for the bug report. I am working on a patch now.

**Comment by jeremylong** [comment]
This issue has been resolved by modifying the Central Analyzer to download the POM file for the given JAR file if it exists in Central and was not found previously by the JAR Analyzer. The issue with the given resolution is that offline use of the CLI may have less accurate results.  Warnings will be added to the documentation.

This issue is resolved in 1.2.9-SNAPSHOT and will be included in a full release soon.
